tests/test_maskedImage.py

```diff
@@ -149,6 +150,55 @@ def testSetGetValues(self):
         self.assertEqual(self.mimage.getMask().get(1, 1), self.EDGE)
         self.assertEqual(self.mimage.getMask().get(2, 2), 0x0)
 
+    def testImagesOverlap(self):
```

Comment:
There's a lot of code duplication with `test_images.py`. Can't some of it (e.g., the bounding box set and the associated subtest) be factored out?